### DISCUSSION TITLE
Add a support to change start menu action

### DISCUSSION THREAD

**Discussion by flourigh** [Flow-Launcher/Flow.Launcher#2597] [Plugin Request]
In default, start button action open a start menu, but, a option to change command like Open Shell

![image](https://github.com/Flow-Launcher/Flow.Launcher/assets/722723/291d5f4a-305f-4986-aeaa-c375671e2d89)

to another personalized command is much powerful

**Comment by EnigmaSolved** [comment]
Have you tried this plugin (which is available within Flow via the Plugin Store)?
https://github.com/AminSallah/Flow.Launcher.Plugin.WinHotkey

**Reply by flourigh** [reply]
Not same, pressing LWin in keyboard and pressing Win in taskbar not same

**Reply by EnigmaSolved** [reply]
Ah, okay, I understand better now what you're getting at.

**Reply by taooceros** [reply]
I suppose we need some kind of very low level hook. None of us is familiar with that so probably until someone is providing help this will not be available in near future.
